provision/infinispan/README.adoc

```diff
@@ -2,55 +2,26 @@
 
 Deploy an Infinispan cluster (with or without cross-site) and configures the necessary caches for Keycloak.
 
-== Required variables
+== Taskfile
 
-* `CLUSTER_1`: The ROSA cluster name of site 1.
-* `CLUSTER_2`: The ROSA cluster name of site 2.
-* `NS_1`: The namespace to install Infinispan in site 1.
-* `NS_2`: The namespace to install Infinispan in site 2.
+For single cluster mode:
 
-== Optional variables:
-* `KUBECONFIG_1`: Path to `KUBECONFIG` to use for CLUSTER_1.
-* `KUBECONFIG_2`: Path to `KUBECONFIG` to use for CLUSTER_2.
-
-If file does not exist, rosa_oc_login will be invoked and the configuration stored in this path.
-
-== Infinispan cluster customization:
-
-* `XSITE_MODE`: The cross-site strategy, default to SYNC.
-* `ISPN_REPLICAS`: The number of Infinispan pods.
-
-== Operation Modes
-
-* CLUSTER_1 == CLUSTER_2 and NS_1 == NS_2 -> Single Infinispan cluster without cross-site.
-
-Example:
-[source, bash]
-----
-CLUSTER_1="gh-pruivo" CLUSTER_2="gh-pruivo" NS_1="ispn-server" NS_2="ispn-server" ./create_ispn_clusters.sh
-----
-
-* CLUSTER_1 == CLUSTER_2 and NS_1 != NS_2 -> Infinispan clusters with cross-site enabled in a single OCP cluster.
-Each namespace gets an Infinispan cluster, and they are linked together with cross-site.
-
-Example:
 [source, bash]
 ----
-CLUSTER_1="gh-pruivo" CLUSTER_2="gh-pruivo" NS_1="server-site-1" NS_2="server-site-2" ./create_ispn_clusters.sh
+task single-cluster ROSA_CLUSTER_NAME=gh-keycloak OC_NAMESPACE=infinispan
 ----
 
-* CLUSTER_1 != CLUSTER_2 -> Infinispan clusters with cross-site enabled in 2 different OCP clusters.
-The namespaces must be set and they can be the same or different namespaces.
+For cross-site between different namespaces:
 
-Example:
 [source, bash]
 ----
-CLUSTER_1="gh-pruivo" CLUSTER_2="gh-keycloak" NS_1="ispn-server" NS_2="ispn-server" ./create_ispn_clusters.sh
+task crossdc-single ROSA_CLUSTER_NAME=gh-keycloak OC_NAMESPACE_1=ispn-1 OC_NAMESPACE_2=ispn-2
```

Comment:
I brought up the ISPN cluster with 

`task crossdc-single ROSA_CLUSTER_NAME=gh-keycloak OC_NAMESPACE_1=ispn-1 OC_NAMESPACE_2=ispn-2` 

@pruivo Once we deployed Keycloak with below configuration, is it possible to switch to a different ISPN namespace in Keycloak i.e. change it to `KC_ISPN_NAMESPACE=ispn-2` and re-deploy. And currently if I do that, I end up in an error state for one of the Keycloak Pods.

```
#ISPN
KC_CUSTOM_INFINISPAN_CONFIG_FILE=config/kcb-infinispan-cache-remote-store-config.xml
KC_ISPN_NAMESPACE=ispn-1
KC_ISPN_CLUSTER=infinispan

#DB
KC_DATABASE=postgres
KC_DB_POOL_MIN_SIZE=15
KC_DB_POOL_MAX_SIZE=15
KC_DB_POOL_INITIAL_SIZE=15
KC_STORAGE=

```

Reply:
```
"message":"Error details:","threadName":"main","threadId":1,"mdc":{},"ndc":"","hostName":"keycloak-2","processName":"QuarkusEntryPoint","processId":1,"exception":{"refId":1,"exceptionType":"java.lang.RuntimeException","message":"Failed to start caches","frames":

[{"class":"org.keycloak.quarkus.runtime.storage.legacy.infinispan.CacheManagerFactory","method":"getOrCreate","line":61},{"class":"org.keycloak.quarkus.runtime.storage.legacy.infinispan.CacheManagerFactory_9497dc12cf1c9738de11a9c4b512d17ae927dceb_Synthetic_ClientProxy","method":"getOrCreate"}
```

Reply:
yup I did use --force flag

Reply:
isn't the cause available in the logs?